docs/src/views/FocusTrapZone.tsx

```diff
@@ -0,0 +1,157 @@
+import * as React from 'react'
+import { Header } from '@stardust-ui/react'
+import DocPage from '../components/DocPage'
+import { link, code } from '../utils/helpers'
+
+import CodeSnippet from '../components/CodeSnippet'
+
+export default () => (
+  <DocPage title="Focus Trap Zone">
+    <Header as="h2">Content</Header>
+    <ul>
+      <li>{link('Overview', '/focus-trap-zone#overview')}</li>
+      <li>{link('Usage', '/focus-trap-zone#usage')}</li>
+      <li>
+        {link(
+          'Override FocusTrapZone settings',
+          '/focus-trap-zone#override-focustrapzone-settings',
+        )}
+      </li>
+    </ul>
+    <Header as="h2">Overview</Header>
+    <p>
+      FocusTrapZone grabs the focus and traps it within an HTML element, usually a dialog or popup.
```

Comment:
for the sake of consistency, lets make `FocusTrapZone` to be formatted as `code` string, the same way it is done for any component mentioned in the guides